cloudbuild/kne_test.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xe
+
+export PATH=${PATH}:/usr/local/go/bin
+gopath=$(go env GOPATH)
+export PATH=${PATH}:$gopath/bin
+
+# Replace exisiting kne repo with new version
+rm -r "$HOME/kne"
+cp -r /tmp/workspace "$HOME/kne"
+
+# Rebuild the kne cli
+pushd "$HOME/kne/kne_cli"
+go build -o kne
+cli="$HOME/kne/kne_cli/kne"
+popd
+
+# Deploy a kind cluster
+pushd "$HOME"
+$cli deploy kne/deploy/kne/kind-bridge.yaml
+
+kubectl get pods -A
```

Comment:
So the only requirement is that kubectl does not exit with an error?

Reply:
I included these mainly for existing in the logs in case a user is debugging a presubmit failure, but in reality they aren't that helpful. I can remove them if you think they add noise. I don't want to validate the state because its not very flexible if dep cfg changes

Reply:
Okay, that is fine then.  You can leave them in.